etc/services/Step 2 - Contextualize.cfg

```diff
@@ -25,7 +25,7 @@
+    "sys_metadata": "{\"GetMobileData\":{\"locX\":292.601023677428,\"locY\":49.587045644821615},\"CalculateAcceleration\":{\"locX\":297.1126479397668,\"locY\":237.35200992161577},\"SendGraphUI\":{\"locX\":168.60200768440336,\"locY\":449.8697628957528},\"SendAcceleration\":{\"locX\":422.34112031773225,\"locY\":450.8283478470718},\"d29b0693-de28-5b3e-9753-65b464106540\":[{\"position\":1,\"target\":\"GetMobileData\",\"left\":0,\"top\":0,\"width\":500,\"align\":\"left\",\"content\":\"Using the same block as the Subscriber in the Step 1- Collect service, this block subscribes to the accelerometer/gyroscope data being published by the niolabs mobile demo UI [[INSERT URL]]. *Connect this block to the modifier block below*.\"},{\"position\":1,\"target\":\"CalculateAcceleration\",\"left\":0,\"top\":0,\"width\":500,\"align\":\"left\",\"content\":\"Using the acceleration readings from the X, Y, and Z axis, this block calculates the overall magnitude of the phones acceleration using the following equation: ```{{ math.sqrt($xAccel ** 2 + $yAccel ** 2 + $zAccel ** 2) / 9.8 }}```. Modifier blocks are just one way to transform a signal into new, useful format in nio. **Connect this block to the modifier block below**.\"},{\"position\":1,\"target\":\"SendAcceleration\",\"left\":0,\"top\":0,\"width\":500,\"align\":\"left\",\"content\":\"This publishes the overall acceleration to the `mobiledemo.accel` topic. We\u2019ll use it in our next service. In order to do so, double click this block to open the block configuration. **Change the topic to `mobiledemo.accel` and click `Accept`.**\"},{\"position\":3,\"target\":\"SendGraphUI\",\"left\":0,\"top\":0,\"width\":500,\"align\":\"left\",\"content\":\"This second publisher sends acceleration values to the `mobiledemo.graph` topic which is displayed on the UI.\"},{\"position\":0,\"target\":null,\"left\":0,\"top\":-200,\"width\":600,\"align\":\"center\",\"content\":\"###Service Overview\\nNow that you\u2019ve seen how a completed nio service looks, let\u2019s get you building! The service below is almost complete. Follow the development notes next to the blocks to complete the final configuration and routing of the service before starting it.\\n\"},{\"position\":0,\"target\":null,\"left\":0,\"top\":550,\"width\":600,\"align\":\"center\",\"content\":\"Start your service if you haven\u2019t done that yet, then go check out the second visualization on the desktop UI. This visualization starts to put a bit more order to the chaos. With this service, we\u2019ve transformed the raw accelerometer data (x, y, & z) into a more useful context--in this case a g-force magnitude. The next service \u201cService 3\u201d will show you an example of triggering an action based on configurable context. Select it in the service panel to continue...\\n\"}]}",
```

Comment:
is the [INSERT URL] placeholder supposed to be here?

Reply:
The second _instruction_ says to _Connect this block to the modifier below_; i think its supposed to connect to the publishers, right?

Also, the previous instruction uses italics, and this one uses bold. They probably could be in their own paragraph.